libr/core/cmd_search.c

```diff
@@ -1924,6 +1924,9 @@ static void do_string_search(RCore *core, struct search_parameters *param) {
 		r_cons_printf ("fs hits\n");
 	}
 	core->search->inverse = param->inverse;
+	if (param->inverse) {
+		core->search->maxhits = 1;
```

Comment:
Just keep the old behaviour. Theres no sense in searching for 0 keywords and ut64max is such a bad number to type for a human to specify that theres no limit